error.spt

```diff
@@ -38,7 +38,7 @@ if code == 500 and not err:
         % endif
         % if sentry_ident
             <p>{{ _(
-                "The details of this error have been recorded. If you decide to contact us "
+                "Details of this error has been recorded. In contacting Liberapay about it, "
```

Comment:
Treating details as a task saves space, complexion. No question raised in second part, just explaining what happens when one does.